pkg/common/events/recorder.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/apache/incubator-yunikorn-k8shim/pkg/conf"
 )
 
-var eventRecorder record.EventRecorder = record.NewFakeRecorder(1024)
+var EventRecorder record.EventRecorder = record.NewFakeRecorder(1024)
```

Comment:
we do not need to expose this if we put the mocked class to the same package, we just need to expose a function to the callers so we can modify the OnXXX functions.

Reply:
got it.